docs/assets/gitbook/custom.css

```diff
@@ -354,3 +354,36 @@
     --c-danger-title: #c00;
     --font-family: "aglet-mono", mono, "divenire-mono", serif, "auger-mono", monospace, "lexia-mono-variable", sans-serif, "moby", sans-serif, "paradroid-mono-soft", sans-serif, "logic-monospace", mono, john-doe, sans-serif, system-ui, -apple-system, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji";
 }
+
+.blog-author-info {
+    padding: 0.25em;
+    text-align: right;
+    font-size: 0.9em;
+    margin-top: -1.25em !important;
+    font-style: italic;
+}
+
+.blog-author-info a {
+    border-bottom: 0;
+    padding: 0;
+}
+
+.blog-author-info a:hover {
+    background: transparent;
+    border-bottom: 0;
+    text-decoration: underline;
+}
+
+a.blog-listing-link {

```

Comment:
I am guilty as charged to treat this CSS file as an “append-only” database.

There are issues to clean it up, and we’ll get there when we get there :).